pubsub/google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -129,4 +129,9 @@ def subscribe(self, subscription, callback=None, flow_control=()):
         subscr = self._policy_class(self, subscription, flow_control)
         if callable(callback):
             subscr.open(callback)
+        else:
+            if callback is not None:
+                string = "{!r} is not callable, please check input" 
```

Comment:
1. Never use `string` as a variable name. Instead here you could say `template` or just avoid the template all together and just do `error = '{!r} ...'.format(callback)`
1. Please use single quotes in your string